### PR TITLE
fix(GradientBlock): fix review issues

### PR DESCRIPTION
**Pull request by fulopdaniel**
1. Add form control to flyout
2. Force gradient points to be between 0 and 100
3. Change some labels

### CODE REVIEW

**Review by ragi96 — CHANGES_REQUESTED** [review]
1. Added a new Gradient on [Product](https://product.frontify.com/document/11#/new-content-blocks/gradient-block) 
2. Toggled "Display CSS Code" on
3. block crashed

Screenshot 2023-04-12 at 09 18 20

File: packages/gradient-block/src/components/ColorFlyout.tsx [hunk 1]
```diff
@@ -124,33 +120,46 @@ export const ColorFlyout = ({
...
-                        validation={colorPositionValidation}
-                    />
+                <div className="tw-w-full tw-pt-5 tw-pl-6 tw-pr-10 tw-flex tw-flex-col tw-gap-6">
+                    <FormControl
+                        label={{
+                            htmlFor: 'GRADIENT_COLOR_COLOR',
+                            children: 'Color',
+                        }}
+                    >
+                        <ColorPickerFlyout
```
[the remark this answers is not in the document]
Comment (fulopdaniel) [reply packages/gradient-block/src/components/ColorFlyout.tsx:130]: Not sure if I understand this one. ColorPickerFlyout is still there, I just wrapped it in a FormControl. Or am I missing something?

**Comment by fulopdaniel** [comment]
> 1. Added a new Gradient on [Product](https://product.frontify.com/document/11#/new-content-blocks/gradient-block)
> 2. Toggled "Display CSS Code" on
> 3. block crashed
> 
> Screenshot 2023-04-12 at 09 18 20

Thanks, great catch. The issues seemed to be some internal version mismatch between code mirror versions. Tweaked it a little bit and now it seems to work. 😵‍💫

**Review by celine-s — APPROVED** [review]
looks good to me 👍

**Review by ragi96 — APPROVED** [review]

**Pull request merged**